src/quo2/components/wallet/account_origin/view.cljs

```diff
@@ -0,0 +1,115 @@
+                       :profile-picture profile-picture}]
+    :recovery-phrase [icons/icon
+                      :i/seed
+                      {:accessibility-label :recovery-phrase-icon
+                       :color               secondary-color}]
+    :private-key     [icons/icon
+                      :i/key
+                      {:accessibility-label :private-key-icon
+                       :color               secondary-color}]
+    :derivation-path [icons/icon
+                      :i/derivated-path
+                      {:accessibility-label :derivation-path-icon
+                       :color               secondary-color}]
+    nil))
+
+(defn- row-view
+  [{:keys [type theme secondary-color profile-picture title stored subtitle on-press]}]
+  [rn/view {:style (style/row-container type theme)}
+   [rn/view {:style style/icon-container}
+    [row-icon profile-picture type secondary-color]]
+   [rn/view
+    {:style style/row-content-container}
+    [row-title type title]
+    [rn/view {:style style/row-subtitle-container}
+     [text/text
+      {:weight :regular
+       :size   :paragraph-2
+       :style  (style/stored-title theme)}
+      subtitle]
+     (when (= :on-keycard stored)
+       [icons/icon
+        :i/keycard-card
+        {:color secondary-color}])]]
+   (when (= :derivation-path type)
+     [rn/pressable
+      {:accessibility-label :derivation-path-button
+       :on-press            on-press
+       :style               style/right-icon-container}
+      [icons/icon
+       :i/options
+       {:color secondary-color}]])])
+
+(defn- list-view
+  [{:keys [type stored profile-picture user-name theme secondary-color]}]
+  (let [stored-name (if (= :on-device stored)
+                      (i18n/label :t/on-device)
+                      (i18n/label :t/on-device))]
+    [row-view
+     {:type            type
+      :stored          stored
+      :profile-picture profile-picture
+      :title           user-name
+      :subtitle        stored-name
+      :theme           theme
+      :secondary-color secondary-color}]))
+
+(defn- card-view
+  [theme derivation-path secondary-color on-press]
+  [row-view
+   {:type            :derivation-path
+    :subtitle        derivation-path
+    :theme           theme
+    :on-press        on-press
+    :secondary-color secondary-color}])
+
+(def view-internal
+  (fn [{:keys [type theme derivation-path on-press] :as props}]
+    (let [secondary-color (colors/theme-colors colors/neutral-50 colors/neutral-40 theme)]
+      [rn/view {:style (style/container theme)}
+       [text/text
+        {:weight :regular
+         :size   :paragraph-2
+         :style  (style/title secondary-color)}
+        (i18n/label :t/origin)]
+       [list-view (assoc props :secondary-color secondary-color)]
+       (when (not= :private-key type)
+         [card-view theme derivation-path secondary-color on-press])])))
+
+(def view
+  "Create an accont-origin UI component.
```

Comment:
typo -> acco`u`nt

Reply:
btw I like this format for the docstrings much easier to read 👍

Reply:
Done, Thank you 😊